Makefile

```diff
@@ -75,6 +75,10 @@ endif
 ci: c
 	./hack/bin/cabal-run-integration.sh $(package)
 
+.PHONY: cabal-fmt
+cabal-fmt:
+	./hack/bin/cabal-fmt.sh $(package)
```

Comment:
I wonder if there is a more automatic way to do this?  Maybe add this goal as a dependency of `default`?

Reply:
Great idea. I've added the formatting to the `c` target as a dependency. Also the script will format all cabal files if invoked with `"all"`, e.g. when running `make c` without a `package=<>`

Reply:
Unfortunately I can't add this to default `make install` target, because that runs on CI, which doesn't have `cabal-fmt` available in its environment